## Publications
### Research Papers

#### Genomics
- **2015-01** | DEEP: a general computational framework for predicting enhancers | *Dimitrios Kleftogiannis,  Panos Kalnis,  Vladimir B. Bajic* | [Nucleic Acids Research](https://academic.oup.com/nar/article/43/1/e6/2902605)
- **2015-03** | DANN: a deep learning approach for annotating the pathogenicity of genetic variants | *Daniel Quang, Yifei Chen, Xiaohui Xie* | [Bioinformatics](https://academic.oup.com/bioinformatics/article/31/5/761/2748191)
- **2015-07** | Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning | *Babak Alipanahi, Andrew Delong, Matthew T Weirauch & Brendan J Frey* | [Nature Biotechnology](https://www.nature.com/articles/nbt.3300)
- **2015-08** | Predicting effects of noncoding variants with deep learning–based sequence model | *Jian Zhou & Olga G Troyanskaya* | [Nature Methods](https://www.nature.com/articles/nmeth.3547)
- **2016-01** | Predicting DNA Methylation State of CpG Dinucleotide Using Genome Topological Features and Deep Networks | *Yiheng Wang, Tong Liu, Dong Xu, Huidong Shi, Chaoyang Zhang, Yin-Yuan Mo & Zheng Wang* | [Scientific Reports](https://www.nature.com/articles/srep19598)
- **2016-02** | Gene expression inference with deep learning | *Yifei Chen, Yi Li, Rajiv Narayan, Aravind Subramanian, Xiaohui Xie* | [Bioinformatics](https://academic.oup.com/bioinformatics/article/32/12/1832/1743989)
- **2016-05** | Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks | *David R. Kelley, Jasper Snoek and John L. Rinn* | [Genome Research](https://genome.cshlp.org/content/26/7/990.short)
- **2016-06** | Convolutional neural network architectures for predicting DNA–protein binding | Haoyang Zeng, Matthew D. Edwards, Ge Liu, David K. Gifford | [Bioinformatics](https://academic.oup.com/bioinformatics/article/32/12/i121/2240609)
- **2016-06** | DanQ: a hybrid convolutional and recurrent deep neural network for quantifying the function of DNA sequences | *Daniel Quang,  Xiaohui Xie* | [Nucleic Acids Research](https://academic.oup.com/nar/article/44/11/e107/2468300#96180312)
- **2016-06** | PEDLA: predicting enhancers with a deep learning-based algorithmic framework | *Feng Liu, Hao Li, Chao Ren, Xiaochen Bo & Wenjie Shu* | [Scientific Reports](https://www.nature.com/articles/srep28517)
- **2017-04** | DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning | *Christof Angermueller, Heather J. Lee, Wolf Reik & Oliver Stegle* | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1189-z)
- **2017-07** | Denoising genome-wide histone ChIP-seq with convolutional neural networks | *Pang Wei Koh, Emma Pierson, Anshul Kundaje* | [Bioinformatics](https://academic.oup.com/bioinformatics/article/33/14/i225/3953958)
- **2017-12** | Predicting enhancers with deep convolutional neural networks | *Xu Min, Wanwen Zeng, Shengquan Chen, Ning Chen, Ting Chen & Rui Jiang* | [BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1878-3)
- **2017** | DEEP MOTIF DASHBOARD: VISUALIZING AND UNDERSTANDING GENOMIC SEQUENCES USING DEEP NEURAL NETWORKS | *JACK LANCHANTIN, RITAMBHARA SINGH, BEILUN WANG and YANJUN QI* | [Pacific Symposium on Biocomputing 2017](https://www.worldscientific.com/doi/abs/10.1142/9789813207813_0025)
- **2018-01** | Deep Learning for Better Variant Calling for Cancer Diagnosis and Treatment | *Anand Ramachandran, Huiren Li, Eric Klee, Steven S. Lumetta, and Deming Chen* | [Proceedings of the 23rd Asia and South Pacific Design Automation Conference](https://dl.acm.org/citation.cfm?id=3201611)
- **2018-05** | Genome-wide prediction of cis-regulatory regions using supervised deep learning methods | *Yifeng Li, Wenqiang Shi & Wyeth W. Wasserman* | [BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2187-1)
- **2018-09** | A universal SNP and small-indel variant caller using deep neural networks | *Ryan Poplin, Pi-Chuan Chang, David Alexander, Scott Schwartz, Thomas Colthurst, Alexander Ku, Dan Newburger, Jojo Dijamco, Nam Nguyen, Pegah T Afshar, Sam S Gross, Lizzie Dorfman, Cory Y McLean & Mark A DePristo* | [Nature Biotechnology](https://www.nature.com/articles/nbt.4235)
- **2018-11** | A deep learning approach to automate refinement of somatic variant calling from cancer sequencing data | *Benjamin J. Ainscough, Erica K. Barnell, Peter Ronning, Katie M. Campbell, Alex H. Wagner, Todd A. Fehniger, Gavin P. Dunn, Ravindra Uppaluri, Ramaswamy Govindan, Thomas E. Rohan, Malachi Griffith, Elaine R. Mardis, S. Joshua Swamidass & Obi L. Griffith* | [Nature Genetics](https://www.nature.com/articles/s41588-018-0257-y)
- **2018-12** | The Unreasonable Effectiveness of Convolutional Neural Networks in Population Genetic Inference | *Lex Flagel, Yaniv Brandvain, Daniel R Schrider* | [Molecular Biology and Evolution](https://academic.oup.com/mbe/article/36/2/220/5229930)
- **2019-03** | A multi-task convolutional deep neural network for variant calling in single molecule sequencing | *Ruibang Luo, Fritz J. Sedlazeck, Tak-Wah Lam & Michael C. Schatz* | [Nature Communications](https://www.nature.com/articles/s41467-019-09025-z)
- **2019-05** | Whole-genome deep-learning analysis identifies contribution of noncoding mutations to autism risk | *Jian Zhou, Christopher Y. Park, Chandra L. Theesfeld, Aaron K. Wong, Yuan Yuan, Claudia Scheckel, John J. Fak, Julien Funk, Kevin Yao, Yoko Tajima, Alan Packer, Robert B. Darnell & Olga G. Troyanskaya* | [Nature Genetics](https://www.nature.com/articles/s41588-019-0420-0)

34. Tan, J., Hammond, J. H., Hogan, D. A. & Greene, C. S. ADAGE-based
integration of publicly available Pseudomonas aeruginosa gene expression data with denoising autoencoders illuminates microbe-host interactions. mSystems 1, e00025-15 (2016).
35. Chen, Y., Li, Y., Narayan, R., Subramanian, A. & Xie, X. Gene expression inference with deep learning. Bioinformatics 32, 1832–1839 (2016).
36. Chen, L., Cai, C., Chen, V. & Lu, X. Learning a hierarchical representation of the yeast transcriptomic machinery using an autoencoder model. BMC Bioinformatics 17 (Suppl. 1), 9 (2016).
37. Cui, H. et al. Boosting gene expression clustering with system-wide biological information: a robust autoencoder approach. Preprint at https://www.biorxiv. org/content/early/2017/11/05/214122 (2017).
38. Xie, R., Wen, J., Quitadamo, A., Cheng, J. & Shi, X. A deep auto-encoder model for gene expression prediction. BMC Genomics 18 (Suppl. 9),
845 (2017).
39. Jha, A., Gazzara, M. R. & Barash, Y. Integrative deep models for alternative splicing. Bioinformatics 33, i274–i282 (2017).
40. Tripathi, R., Patel, S., Kumari, V., Chakraborty, P. & Varadwaj, P. K. DeepLNC, a long non-coding RNA prediction tool using deep neural network. Netw. Model. Anal. Health Inform. Bioinform. 5, 21 (2016).
41. Yu, N., Yu, Z. & Pan, Y. A deep learning method for lincRNA detection using auto-encoder algorithm. BMC Bioinformatics 18 (Suppl. 15), 511 (2017).
42. Hill, S. T. et al. A deep recurrent neural network discovers complex biological rules to decipher RNA protein-coding potential. Nucleic Acids Res. 46, 8105–8113 (2018).
43. Angermueller, C., Lee, H. J., Reik, W. & Stegle, O. DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning. Genome Biol. 18, 67 (2017).
44. Shaham, U. et al. Removal of batch effects using distribution-matching residual networks. Bioinformatics 33, 2539–2546 (2017).
45. Lin, C., Jain, S., Kim, H. & Bar-Joseph, Z. Using neural networks for reducing the dimensions of single-cell RNA-Seq data. Nucleic Acids Res. 45, e156 (2017).
46. Poplin, R. et al. Creating a universal SNP and small indel variant caller with deep neural networks. Preprint at https://www.biorxiv.org/content/ early/2018/03/20/092890 (2017).
47. Luo, R., Sedlazeck, F.J., Lam, T.-W. & Schatz, M. Clairvoyante: a multi-task convolutional deep neural network for variant calling in single molecule sequencing. Preprint at https://www.biorxiv.org/content/ early/2018/09/26/310458 (2018).
48. Luo, R., Lam, T.-W. & Schatz, M. Skyhawk: an artificial neural network-based discriminator for reviewing clinically significant genomic variants. Preprint at https://www.biorxiv.org/content/early/2018/05/01/311985 (2018).
49. Torracinta, R. et al. Adaptive somatic mutations calls with deep learning and semi-simulated data. Preprint at https://www.biorxiv.org/content/ early/2016/10/04/079087 (2016).
50. Boža, V., Brejová, B. & Vinař, T. DeepNano: deep recurrent neural networks for base calling in MinION nanopore reads. PLoS One 12, e0178751 (2017).
51. Teng, H., Hall, M.B., Duarte, T., Cao, M.D. & Coin, L. Chiron: translating nanopore raw signal directly into nucleotide sequence using deep learning. Preprint at https://www.biorxiv.org/content/early/2017/08/23/179531 (2017).
52. Qi, H. et al. MVP: predicting pathogenicity of missense variants by deep neural networks. Preprint at https://www.biorxiv.org/content/ early/2018/02/02/259390 (2018).
53. Quang, D., Chen, Y. & Xie, X. DANN: a deep learning approach for annotating the pathogenicity of genetic variants. Bioinformatics 31, 761–763 (2015).
54. Korvigo, I., Afanasyev, A., Romashchenko, N. & Skoblov, M. Generalising better: applying deep learning to integrate deleteriousness prediction scores for whole-exome SNV studies. PLoS One 13, e0192829 (2018).
55. Yuan, Y. et al. DeepGene: an advanced cancer type classifier based on deep learning and somatic point mutations. BMC Bioinformatics 17, 476 (2016).
56. Yousefi, S. et al. Predicting clinical outcomes from large scale cancer genomic profiles with deep survival models. Sci. Rep. 7, 11707 (2017).
57. Ma, W., Qiu, Z., Song, J., Cheng, Q. & Ma, C. DeepGS: predicting phenotypes from genotypes using deep learning. Preprint at https://www.biorxiv.org/ content/early/2017/12/31/241414 (2017).
58. Zhou, J. et al. Whole-genome deep learning analysis reveals causal role of noncoding mutations in autism. Preprint at https://www.biorxiv.org/content/ early/2018/05/11/319681 (2018).
59. Zhou, J. et al. Deep learning sequence-based ab initio prediction of variant effects on expression and disease risk. Nat. Genet. 50, 1171–1179 (2018).
60. Sundaram, L. et al. Predicting the clinical impact of human mutation with deep neural networks. Nat. Genet. 50, 1161–1170 (2018).

### Reviews and Perspectives
#### General
- **2015-05** | Deep Learning | *Yann LeCun, Yoshua Bengio & Geoffrey Hinton* | [Nature](https://www.nature.com/articles/nature14539)
- **2016-07** | Deep learning for computational biology | *Christof Angermueller, Tanel Pärnamaa, Leopold Parts, Oliver Stegle* | [Molecular Systems Biology](https://www.embopress.org/doi/full/10.15252/msb.20156651)
- **2017-01** | Deep learning for health informatics | *Daniele Ravì, Charence Wong, Fani Deligianni, Melissa Berthelot, Javier Andreu-Perez, Benny Lo, and Guang-Zhong Yang* | [IEEE journal of biomedical and health informatics](https://ieeexplore.ieee.org/abstract/document/7801947)
- **2017-12** | Deep learning for health informatics: Recent trends and future directions | *Srivastava, Siddharth, Sumit Soman, Astha Rai, and Praveen K. Srivastava* | [2017 International Conference on Advances in Computing, Communications and Informatics (ICACCI)](https://ieeexplore.ieee.org/abstract/document/8126082)
- **2018-04** | Opportunities and obstacles for deep learning in biology and medicine | *Travers Ching, Daniel S. Himmelstein, Brett K. Beaulieu-Jones, Alexandr A. Kalinin, Brian T. Do, Gregory P. Way, Enrico Ferrero, Paul-Michael Agapow, Michael Zietz, Michael M. Hoffman, Wei Xie, Gail L. Rosen, Benjamin J. Lengerich, Johnny Israeli, Jack Lanchantin, Stephen Woloszynek, Anne E. Carpenter, Avanti Shrikumar, Jinbo Xu, Evan M. Cofer, Christopher A. Lavender, Srinivas C. Turaga, Amr M. Alexandari, Zhiyong Lu, David J. Harris, Dave DeCaprio, Yanjun Qi, Anshul Kundaje, Yifan Peng, Laura K. Wiley, Marwin H. S. Segler, Simina M. Boca, S. Joshua Swamidass, Austin Huang, Anthony Gitter  and Casey S. Greene* | [Journal of The Royal Society Interface](https://royalsocietypublishing.org/doi/full/10.1098/rsif.2017.0387)
- **2018-06** | Next-Generation Machine Learning for Biological Networks | *Diogo M.Camacho, Katherine M.Collins, Rani K.Powers, James C.Costello, James J.Collins* | [Cell] (https://www.sciencedirect.com/science/article/pii/S0092867418305920)
- **2018-09** | Deep learning in biomedicine | *Michael Wainberg, Daniele Merico, Andrew Delong & Brendan J Frey* | [Nature Biotechnology](https://www.nature.com/articles/nbt.4233)
- **2019-01** | A guide to deep learning in healthcare | *Andre Esteva, Alexandre Robicquet, Bharath Ramsundar, Volodymyr Kuleshov, Mark DePristo, Katherine Chou, Claire Cui, Greg Corrado, Sebastian Thrun & Jeff Dean* | [Nature Medicine](https://www.nature.com/articles/s41591-018-0316-z)
- **2019-01** | Guidelines for reinforcement learning in healthcare | *Omer Gottesman, Fredrik Johansson, Matthieu Komorowski, Aldo Faisal, David Sontag, Finale Doshi-Velez & Leo Anthony Celi* | [Nature Medicine](https://www.nature.com/articles/s41591-018-0310-5)
- **2019-03** | Reinforcement learning in artificial and biological systems | *Emre O. Neftci and Bruno B. Averbeck* | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-019-0025-4.pdf?origin=ppub)

#### Genomics
- **2016-01** | Machine learning in genomic medicine: a review of computational problems and data sets | *Michael KK Leung, Andrew Delong, Babak Alipanahi, and Brendan J Frey* | [Proceedings of the IEEE](https://ieeexplore.ieee.org/abstract/document/7347331)
- **2018-05** | Deep learning of genomic variation and regulatory network data | *Amalio Telenti, Christoph Lippert, Pi-Chuan Chang, Mark DePristo* | [Human Molecular Genetics](https://academic.oup.com/hmg/article-abstract/27/Supplement_R1/R63/4966854)
- **2018-05** | Deep Learning for Genomics: A Concise Overview | *Tianwei Yue, Haohan Wang* | [Invited chapter for Springer Book: Handbook of Deep Learning Applications](https://arxiv.org/abs/1802.00810)
- **2019-01** | A primer on deep learning in genomics | *James Zou, Mikael Huss, Abubakar Abid, Pejman Mohammadi, Ali Torkamani and Amalio Telenti* | [Nature Genetics](https://www.nature.com/articles/s41588-018-0295-5) | [Google Colab Notebook with tutorial](https://colab.research.google.com/drive/17E4h5aAOioh5DiTo7MZg4hpL6Z_0FyWr)
- **2019-04** | Deep learning: new computational modelling techniques for genomics| *Gökcen Eraslan, Žiga Avsec, Julien Gagneur & Fabian J. Theis* | [Nature Reviews Genetics](https://www.nature.com/articles/s41576-019-0122-6)
